### PR TITLE
Use the WA DOR tax rate API

### PR DESCRIPTION
**Pull request by jeremyfelt**
This gives us real time tax rates and removes the need to maintain a massive, possibly inaccurate table.

Fixes https://github.com/washingtonstateuniversity/wsupress.wsu.edu/issues/32

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: includes/sales-tax.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+	if ( 'WA' !== $state ) {
+		return array();
+	}
+
+	$matched_rates = array();
+
+	$tax_url = 'http://dor.wa.gov/AddressRates.aspx?output=text';
+	$tax_url = add_query_arg( array(
+		'output' => 'text',
```
Comment (philcable) [inline includes/sales-tax.php]: The `output=text` parameter is already part of `$tax_url` above, though having it twice doesn't seem to hurt anything :)

**Review by philcable — APPROVED** [review]
👍  Looks good! I left an inline comment about one simple thing.

File: includes/sales-tax.php (hunk 1 above)
Comment (jeremyfelt) [reply includes/sales-tax.php]: oh, duh! I'll fix that :)

**Pull request merged**